repobee_sanitizer/sanitizer.py

```diff
@@ -96,12 +106,47 @@ def command(self) -> Optional[plug.Result]:
                 status=plug.Status.ERROR,
             )
 
+        result_message = "Successfully sanitized repo" + (
+            f" to pull request branch\n\nrun 'git switch "
+            f"{effective_target_branch}' to checkout the branch"
+            if self.create_pr_branch
+            else ""
+        )
+
```

Comment:
Dont like how this just appears, might be worth putting into a function and then placing the `msg` field in the result with the function call?

Reply:
I can add the reasoning for doing this.

The alternative is calling a separate function, however, calling it with `effective_target_branch` will crash the tests running no-commit, so there's no super clean way to solve that which I can think of. 

defining an empty `effective_target_branch` at the start of the `command` function works but that feels weird when it's not always used, specifically for running no-commit.

instead, assigning with `self.effective_target_branch` works, then we don't have to call the function with an argument, and we can check `if create_pr_branch` inside the function, but that also feels like a hacked solution.

Reply:
This is also based on if we want to report the name of the created branch or not